script/run-unit-test.js

```diff
@@ -27,6 +28,7 @@ if (
     '/src/',
     `/src/applications/${options['app-folder']}/`,
   );
+  coverageInclude = `--include 'src/applications/${options['app-folder']}/**'`;
```

Comment:
I added this so that if you use `--coverage` and `--app-folder` at the same time, you get coverage results that are constrained to just your app folder.